<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LAB</class>
 <widget class="QWidget" name="LAB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,4">
     <item>
      <widget class="QLabel" name="label_WARN">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_IMG">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 87, 83);</string>
       </property>
       <property name="text">
        <string>IMG</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_ID">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_TIME">
         <property name="text">
          <string>TIME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_CAMID">
         <property name="text">
          <string>CAMID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_STATUS">
         <property name="text">
          <string>STATUS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
